Add initial PR for Contributing & Coding Standard (#920)
community-scripts/ProxmoxVE: .github/CONTRIBUTOR_GUIDE/json/AppName.md

# **AppName<span></span>.json Files**

>  `AppName.json` files found in the `/json` directory. These files are used to provide informations for the frontend. For this guide we take `/json/snipeit.json` as example.

Reviewer: ```suggestion
 `AppName.json` files found in the `/json` directory. These files are used to provide informations for the website. For this guide we take `/json/snipeit.json` as example.
```

'Frontend' is too specific. Website is easier to understand.

>  ## 1 **Json Generator**
>  - To spare you some headache creating the json file, use the [Json-Editor](https://community-scripts.github.io/ProxmoxVE/json-editor)

Reviewer: ```suggestion
 ## 1. JSON Generator
Use the [JSON Generator](https://community-scripts.github.io/ProxmoxVE/json-editor) to create this file for your application.
```
